examples/seq2seq/finetune_trainer.py

```diff
@@ -252,45 +293,56 @@ def main():
         data_args=data_args,
     )
 
+    all_metrics = {}
     # Training
     if training_args.do_train:
+        logger.info("*** Train ***")
+
+        start_time = time.time()
         trainer.train(
             model_path=model_args.model_name_or_path if os.path.isdir(model_args.model_name_or_path) else None
         )
+        metrics = speed_metrics("train", start_time, data_args.n_train)
+
         trainer.save_model()
```

Comment:
This line also saves the tokenizer, so maybe we can add a comment before it to say: "Oh, by the way, this also saves the tokenizer."